client/src/components/admin/RoleForm.vue

```diff
@@ -27,9 +27,6 @@
                             :multiple="true"
                             :optional="true" />
                     </FormElementLabel>
-                    <FormElementLabel title="Create a new group of the same name for this role:">
```

Comment:
You could just do that sequentially on the frontend. But it's easy enough to have the user do that separately.